mattermost-helm/templates/config.tpl

```diff
@@ -309,14 +309,22 @@
         "TurnUsername": "",
         "TurnSharedKey": ""
     },
+    "PluginSettings": {
+        "Enable": true,
+        "EnableUploads": true,
+        "Directory": "./plugins",
+        "ClientDirectory": "./client/plugins",
+        "Plugins": {},
+        "PluginStates": {}
+    },
     "DataRetentionSettings": {
         "Enable": false
     }
     {{- if .Values.global.features.jobserver.enabled -}}
     ,
     "JobSettings": {
-        "RunJobs": false,
-        "RunScheduler": false
+        "RunJobs": true,
```

Comment:
There is a separate job server so shouldn't these be disabled?

Reply:
ahhh got it